### PR TITLE
Downloader allow headers

### PR DESCRIPTION
**Pull request by qparis**
A small patch also required for GoG. It allows to set headers to Download bean

### CODE REVIEW

File: phoenicis-tools/src/main/java/org/phoenicis/tools/http/Downloader.java [hunk 1]
```diff
@@ -122,8 +170,24 @@ public String get(URL url, Consumer<ProgressEntity> onChange) {
      * @param onChange consumer to show the download progress (e.g. a progress bar)
      */
     private void get(URL url, OutputStream outputStream, Consumer<ProgressEntity> onChange) {
+        get(url, outputStream, null, onChange);
```
[the remark this answers is not in the document]
Comment (qparis) [reply phoenicis-tools/src/main/java/org/phoenicis/tools/http/Downloader.java]: I think it is safer to support null anyway, so it does not change anything

**Comment by madoar** [comment]
So basically this PR adds header support to the `Downloader`?
Everything is else is part of #1681?

**Comment by qparis** [comment]
> So basically this PR adds header support to the `Downloader`?
> Everything is else is part of #1681?

Yes

**Comment by qparis** [comment]
Yes, the change is backward compatible

**Comment by plata** [comment]
Ok. Please merge master to fix the diff.

**Comment by qparis** [comment]
I messed up... Sorry

**Pull request closed without merging**